src/components/program-progress/ProgramProgressPage.jsx

```diff
@@ -37,9 +37,10 @@ const ProgramProgressPage = () => {
     [program],
   );
   const courseData = program?.data?.courseData;
-  const totalCoursesInProgram = courseData?.notStarted?.length
-    + courseData?.completed?.length
-    + courseData?.inProgress?.length;
+  const notStartedCourses = courseData?.notStarted?.length;
+  const completedCourses = courseData?.completed?.length;
+  const inProgressCourses = courseData?.inProgress?.length;
```

Comment:
Would we want to fallback to `0` if `notStarted`, `completed`, or `inProgress` is undefined/null? E.g.

```jsx
const notStartedCourses = courseData?.notStarted?.length || 0;
const completedCourses = courseData?.completed?.length || 0;
const inProgressCourses = courseData?.inProgress?.length || 0;
const totalCoursesInProgram = notStartedCourses + completedCourses + inProgressCourses;
```

Otherwise, I think we'll be adding undefined/nulls together.